app/src/main/java/protect/card_locker/CatimaAppCompatActivity.java

```diff
@@ -20,6 +21,15 @@ protected void attachBaseContext(Context base) {
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        if (Build.VERSION.SDK_INT >= 23) {
+            if (AppCompatDelegate.getDefaultNightMode() == AppCompatDelegate.MODE_NIGHT_NO
```

Comment:
What happens if the app is in dark mode? can you not change the color?

Reply:
The color change is only done after you picked your theme in the app's settings page. This piece of code is checking the settings of the app at that point to apply the appropriate theme.

Also, Catima defaults navbar color to black so that's why I did not explicitly do an else-if block for it.